### DISCUSSION TITLE
Web interface Lay-Z-spa

### DISCUSSION THREAD

**Discussion by snickerit** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#108] [Q&A]
Hi.
Where can I find the code for the Web interface "spa controll"?
Can not find "6-wire version", open folder.

**Comment by visualapproach** [comment]
Hi @snickerit 
Don't know if this is what you are asking, but download the zip-file as the image below. Unzip it and there is the code :-)
If you are adventurous and want the latest and greatest, you may try the development branch.
![image](https://user-images.githubusercontent.com/22987250/130207567-3f82b5af-61ef-4ca0-a71b-4ba90250efeb.png)
